Protos/V1/tasks_service.proto

```diff
@@ -0,0 +1,159 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "objects.proto";
+import "task_status.proto";
+
+package armonik.api.grpc.v1.tasks;
+
+option csharp_namespace = "Armonik.Api.Grpc.V1.Tasks";
+
+/**
+ * The TaskService provides methods for listing and getting tasks.
+ */
+service tasks {
+  // Get all tasks with pagination
+  rpc list(ListTasksRequest) returns (ListTasksResponse) {}
+
+  // Get a task by id
+  rpc get(GetTaskRequest) returns (GetTaskResponse) {}
+}
+
+/**
+ * A raw task object
+ */
+message TaskRaw {
+  string id = 1;
+  string session_id = 2;
+  string owner_pod_id = 3;
+
+  repeated string parent_task_ids = 4;
+  repeated string data_dependencies = 5;
+  repeated string expected_output_ids = 6;
+  repeated string retry_if_ids = 7;
+
+  TaskStatus.TaskStatus status = 8;
+  string status_message = 9;
+
+  TaskOptions.TaskOptions options = 10;
```

Comment:
After we updated the packages names to lower case this should be referenced as task_status.TaskStatus

Reply:
Yes, will change this